notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    CAPTION = "a man &lt;|image|&gt; and a man &lt;|image|&gt; are reading book together"
</pre><p>for what CAPTION and DEMO_NAME varibales used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Removed